src/contract_wrappers/tokens/security_token_wrapper/common.ts

```diff
@@ -2268,6 +2265,37 @@ export default abstract class SecurityTokenCommon extends ERC20TokenWrapper {
     return bytes32ArrayToStringArray(await (await this.contract).getAllDocuments.callAsync());
   };
 
+  // public signTransferData = async (txData: Partial<TxData> | undefined): Promise<string> => {
```

Comment:
could you remove this comment or it's for some reason?

Reply:
Yeah